examples/receiver-get-200.json

```diff
@@ -0,0 +1,4 @@
+[
+  "sinks/",
+  "edid/"
```

Comment:
moved?

Reply:
I fixed it in 91bd52b080ec3d71f6af658c2a340a64fd8fc09e. Thank you.